### ISSUE TITLE
ITV player stops after ads are blocked

### ISSUE THREAD

**Issue by Fit4GHub** [uBlockOrigin/uAssets#24613]
### Prerequisites

- [X] I verified that this is not a filter list issue. Report any issues with filter lists or broken website functionality in the [uAssets issue tracker](https://github.com/uBlockOrigin/uAssets/issues).
- [X] This is **NOT** a [_YouTube_](https://www.github.com/uBlockOrigin/uAssets/issues/20586), [_Facebook_](https://www.github.com/uBlockOrigin/uAssets/issues/3367) or [_Twitch_](https://www.github.com/uBlockOrigin/uAssets/issues/5184) report. _These sites MUST be reported by clicking their **respective links**_.
- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uBlock-issues/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] The issue is not present after disabling uBO in the browser.
- [X] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I am reporting is not normal behavior.

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

Hi recent fix has blocked the ads but now the player stops after the ads are blocked and only way to restart the stream is by refreshing the page. Once done the player restarts and plays on as if the ads had finished running.

I've searched github etc and this seems to be a new issue.

### A specific URL where the issue occurs.

```yaml
https://www.itv.com/
```

### Steps to Reproduce

navigate to site and try a program

### Expected behavior

prog plays and skips ads at each ad break moving onto the next section of program

### Actual behavior

prog plays and stops at ad break, will not moving onto the next section of program unless page is manually refreshed

### uBO version

latest, auto updates filters etc

### Browser name and version

Firefox 128

### Operating System and version

 win 10

**Comment by Fit4GHub** [comment]
This is not a duplicate issue, have you checked the player. It now stops
after ads are blocked. Before the ads weren't being blocked at all

**Comment by stephenhawk8054** [comment]
Just continue in the old thread

**Comment by Fit4GHub** [comment]
I'll comment in the original thread 24517 if that helps.

**Comment by Fit4GHub** [comment]
Yep done. Sorry for starting a new thread unnecessarily, I'm new to posting
and didn't realise you could update a closed thread with a new issue.

Many thanks.

**Comment by Fit4GHub** [comment]
Also apologies for starting a new thread unnecessarily, I'm new to posting
and didn't realise you could update a closed thread with a new issue.

Many thanks.
